src/TKey.cpp

```diff
@@ -72,12 +74,15 @@ void TKey::setName(const QString& name)
     mpHost->getKeyUnit()->mLookupTable.insertMulti(name, this);
 }
 
-bool TKey::match(int key, int modifier, const bool isToMatchAll)
+bool TKey::match(const int key, const Qt::KeyboardModifiers modifiers, const bool isToMatchAll)
 {
     bool isAMatch = false;
     if (isActive()) {
         if (!isFolder()) {
-            if ((mKeyCode == key) && (mKeyModifier == modifier)) {
+            if (  (mKeyCode == key) // The key matches
+               && ((modifiers & mPresentModifiers) == mPresentModifiers) // All of the required present modifiers are present and thus this matches
+               && (!(modifiers & mAbsentModifiers))) { // None of the required to be absent modifiers are present and thus this matches
+
```

Comment:
This blank line is one of the things that the restyle bot is moaning about. As a matter of coding practice when the conditions for an `if` are complex ***and take more than one line to state*** I always leave a blank line between them and the conditionally executed code that follows. It helps my eyes to pick out the condition better.

Is there a clang-format setting for this - and can we discuss whether this is not an unreasonable styling option?

Reply:
I've turned the bot off, because it doesn't understand `clang-format`. I'll personally never add blank lines after `if` conditions but the bot shouldn't complain on them either.